package.json

```diff
@@ -1,90 +1,90 @@
+        "down": "yarn sequelize-cli db:migrate:undo",
+        "seed": "yarn sequelize-cli db:seed:all",
+        "unseed": "yarn sequelize-cli db:seed:undo:all",
+        "create:roles": "yarn sequelize db:seed --seed 20220411084040-Role.js",
+        "create:admin": "yarn run build && node lib/config/init.js",
+        "build": "babel src -d lib",
+        "start": "yarn migrate --env production && cross-env NODE_ENV=production yarn create:admin && cross-env NODE_ENV=production node lib/index.js",
+        "dev": "yarn migrate && yarn create:admin && cross-env NODE_ENV=development nodemon --exec babel-node src/index.js",
+        "lint": "eslint src/** test/** --fix",
+        "prepare": "husky install",
+        "test": "yarn migrate --env test && cross-env NODE_ENV=test c8 mocha --timeout 1000000 --exit --require @babel/register"
+    },
+    "dependencies": {
+        "@babel/cli": "^7.17.6",
+        "@babel/core": "^7.8.4",
+        "@babel/eslint-parser": "^7.17.0",
+        "@babel/node": "^7.8.4",
+        "@babel/plugin-transform-runtime": "^7.17.0",
+        "@babel/preset-env": "^7.8.4",
+        "@babel/runtime": "^7.17.8",
+        "babel-plugin-istanbul": "^6.0.0",
+        "babel-plugin-transform-async-to-generator": "^6.24.1",
+        "babel-plugin-transform-runtime": "^6.23.0",
+        "babel-preset-latest": "^6.24.1",
+        "bcrypt": "^5.0.1",
+        "c8": "^7.11.0",
+        "chai": "^4.3.6",
+        "chai-http": "^4.3.0",
+        "cloudinary": "^1.29.1",
+        "core-js": "^3.21.1",
+        "cors": "^2.8.5",
+        "cross-env": "6",
+        "datauri": "^4.1.0",
+        "deep-email-validator": "^0.1.21",
+        "dotenv": "^16.0.0",
+        "ejs": "^3.1.6",
+        "express": "^4.17.3",
+        "express-fileupload": "^1.3.1",
+        "express-session": "^1.17.2",
+        "googleapis": "^99.0.0",
+        "ioredis": "^5.0.3",
+        "joi": "^17.6.0",
+        "jsonwebtoken": "^8.5.1",
+        "mocha": "^9.2.2",
+        "multer": "^1.4.4",
+        "nodemailer": "^6.7.3",
+        "nyc": "^15.1.0",
+        "passport": "^0.5.2",
+        "passport-facebook": "^3.0.0",
+        "passport-google-oauth2": "^0.2.0",
+        "pg": "^8.7.3",
+        "pg-hstore": "^2.3.4",
+        "sequelize": "^6.17.0",
+        "sequelize-cli": "^6.4.1",
+        "sequelize-replace-enum-postgres": "^1.6.0",
+        "supertest": "^6.2.2",
+        "swagger-jsdoc": "^6.1.0",
+        "swagger-ui-express": "^4.3.0",
+        "uuid": "^8.3.2"
+    },
+    "devDependencies": {
+        "@babel/register": "^7.8.3",
+        "@sinonjs/referee": "^9.1.1",
+        "eslint": "8.1.0",
+        "eslint-config-airbnb-base": "^15.0.0",
+        "eslint-config-prettier": "^8.5.0",
+        "eslint-plugin-import": "^2.25.4",
+        "eslint-plugin-prettier": "^4.0.0",
+        "husky": "^7.0.0",
+        "istanbul": "^0.4.5",
+        "nodemon": "^2.0.15",
+        "prettier": "^2.6.0",
+        "sinon": "^13.0.1"
+    }
+}
```

Comment:
unnecessary indentation changes. make sure you remove the/ vs-code folder and use prettier as per this project configuration